internal/operators/ocs/validations.go

```diff
@@ -74,11 +53,6 @@ func (o *ocsValidator) ValidateRequirements(cluster *models.Cluster) (api.Valida
 
 	if int64(len(hosts)) < o.OCSRequiredHosts {
 		status = "Insufficient hosts to deploy OCS. A minimum of 3 hosts is required to deploy OCS. "
-		err = setOperatorStatus(cluster, status)
-		if err != nil {
-			o.log.Error("Failed to set Operator status ", err)
-			return api.Failure, "Failed to set Operator status "
-		}
 		o.log.Info("Setting Operator Status ", status)
```

Comment:
please change the log to: "Validate Requirements status:"

Reply:
fixed